src/legacy/core_plugins/metrics/public/components/vis_editor.js

```diff
@@ -174,6 +180,12 @@ class VisEditor extends Component {
   componentDidUpdate() {
     this.props.renderComplete();
   }
+
+  componentWillUnmount() {
+    if (this.updateVisState.cancel) {
```

Comment:
👍 pretty sure there's no reason to check for cancel first though, if it was missing then something really crazy would be wrong.

Reply:
np, removed